### ISSUE TITLE
How to use DrawSVGPlugin?

### ISSUE THREAD

**Issue by xx4159** [greensock/GreenSock-JS#283]
I wanna use DrawSVGPlugin.
Is there any way to use it?

**Comment by jackdoyle** [comment]
Absolutely! DrawSVGPlugin is a membership benefit of Club GreenSock: https://greensock.com/club/. When you sign up, just go to your account dashboard and download the zip which has the bonus files too. 

Enjoy!
